### ISSUE TITLE
Top menu/banner covers section header

### ISSUE THREAD

**Issue by vreuter** [databio/mkdocs-databio#16]
When navigating to a specific header via link in docs, a docs site's top menu / banner can cover up the section header. Not sure if this is arises from this package, or from configuration specific to packages that build docs with this package, but I'm opening this here as a best-guess. The example is the class hierarchy docs from `attmap`

**Comment by nsheff** [comment]
I think that's a cache issue... confirm that this doc:

view-source:http://attmap.databio.org/en/latest/css/extra.css

says at the top: 

```
body {
  padding-top: 56px;
}
```

**Comment by vreuter** [comment]
> I think that's a cache issue... confirm that this doc:
> 
> view-source:http://attmap.databio.org/en/latest/css/extra.css
> 
> says at the top:
> 
> ```
> body {
>   padding-top: 56px;
> }
> ```

Indeed it does.

**Comment by nsheff** [comment]
if you refresh the page does it solve it?

**Comment by vreuter** [comment]
No, but this is also from a local `serve` command, rather than anything actually built and viewed in a more standard way, so it could well be specific to my use case.

**Comment by vreuter** [comment]
Sorry, meant to post the screenshot earlier for clarity...

![attmap_API_docs_hidden_header](https://user-images.githubusercontent.com/12036370/59064928-7bbe4c00-8879-11e9-9725-fc2c2437c385.png)

**Comment by nsheff** [comment]
oh. I misinterpreted your comment, sorry. Now I understand.

**Comment by vreuter** [comment]
Sorry, my bad--like said, I took the screenshot and then forgot to post it.

**Comment by nsheff** [comment]
This is an issue with links and anchors. I've run into stuff like this before. Not sure how to fix.
